src/AddTaskPage.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+
+export function AddTaskPage(props) {
+
+    return (
+        <form onSubmit={(e) => props.addNewDataPoint(e, document.getElementById("newTask").value)}>
```

Comment:
No particular need to have a form.

You could just have an input type="button" that has an onClick method.